src/ign_TEST.cc

```diff
@@ -1795,6 +1795,50 @@ TEST(GraphCmd, IGN_UTILS_TEST_DISABLED_ON_WIN32(ModelFrameAttachedTo))
   EXPECT_EQ(sdf::trim(expected.str()), sdf::trim(output));
 }
 
+/////////////////////////////////////////////////
+TEST(inertial_stats, IGN_UTILS_TEST_DISABLED_ON_WIN32(SDF))
+{
+  std::string pathBase = PROJECT_SOURCE_PATH;
+  pathBase += "/test/sdf";
+
+  auto expectedOutput =
+    std::string("Inertial statistics for model: test_model\n") +
+    std::string("---\n") +
+    std::string("Total mass of the model: 24\n") +
+    std::string("---\n") +
+    std::string("Centre of mass in model frame: \n") +
+    std::string("X: 0\n") +
+    std::string("Y: 0\n") +
+    std::string("Z: 0\n") +
+    std::string("---\n") +
+    std::string("Moment of inertia matrix: \n") +
+    std::string("304  0    0    \n") +
+    std::string("0    304  0    \n") +
+    std::string("0    0    604  \n") +
+    std::string("---\n");
+
+  // Check a good SDF file by passing the absolute path
+  {
+    std::string path = pathBase +"/inertial_stats.sdf";
+
+    std::string output =
+      custom_exec_str(IgnCommand() + " sdf --inertial-stats " +
+                      path + SdfVersion());
+    EXPECT_EQ(expectedOutput, output);
+  }
+
+  // Check a good SDF file from the same folder by passing a relative path
+  {
+    std::string path = "inertial_stats.sdf";
+
+    std::string output =
+      custom_exec_str("cd " + pathBase + " && " +
+                      IgnCommand() + " sdf --inertial-stats " +
+                      path + SdfVersion());
+    EXPECT_EQ(expectedOutput, output);
+  }
```

Comment:
I don't think this test is necessary but it would be good to test some invalid cases such as a world file and/or [`sdformat/test/sdf/inertial_invalid.sdf`](https://github.com/ignitionrobotics/sdformat/blob/4dd71a5943d438bb5cff10a3206c3c394a5ef453/test/sdf/inertial_invalid.sdf)

Reply:
Done